docs/nuget-org/nuget-org-faq.md

```diff
@@ -59,6 +59,17 @@ It is possible to share dependencies between multiple packages, making the total
 
 Dependencies are mostly static and never change. When fixing a bug in code, the dependencies may not need to be updated. If you bundle dependencies, you end up reshipping larger packages every time. By splitting NuGet packages into related dependencies, upgrades are much more fine-grained for consumers of your package.
 
+**How do I change my package ID casing?**
+
+NuGet.org displays the package's original casing (see [NuGetGallery#3349](https://github.com/NuGet/NuGetGallery/issues/3349)). If you'd like to change your package ID from `foo.bar` to `Foo.Bar`, please follow these steps:
+
+1. Ensure that the latest version of your package on nuget.org has the desired package ID casing. If needed, please upload a new version of your package with the desired casing.
```

Comment:
Nit: do we have a standing on nuget.org vs. NuGet.org in our docs? I see some inconsistency in general but perhaps we can do our best for new changes.

Reply:
Maybe add a line to update the issue so that we can prioritize the bug fix appropriately?

Reply:
There's no stance on nuget.org vs. NuGet.org. both are valid. Use of proper name is preferred, but common name is more practical.

Reply:
Oops the inconsistency in my section was unintentional. This page uses NuGet.org, I meant to be consistent with that.